.ci/install_cri_containerd.sh

```diff
@@ -69,6 +70,15 @@ cat << EOF | sudo tee  /etc/systemd/system/kubelet.service.d/0-containerd.conf
 Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
```

Comment:
I agree with your commit and note this is not part of your change, but while I'm here...  @jcvenegas @chavafg I question configuring kubernetes in a containerd installation script?  Seems that should be done elsewhere?

Reply:
Hey @egernst , It is done at k8s setup, depending on a job flag, will decide to pass the socket for containerd or cri-o.

Reply:
https://github.com/kata-containers/tests/blob/master/integration/kubernetes/init.sh#L52